packages/output/src/common/output-channel.ts

```diff
@@ -20,29 +20,29 @@ import URI from '@theia/core/lib/common/uri';
 import { Deferred } from '@theia/core/lib/common/promise-util';
 import { Resource, ResourceResolver } from '@theia/core/lib/common/resource';
 import { CommandRegistry, CommandContribution } from '@theia/core/lib/common/command';
-import { OpenerService, open, QuickPickService } from '@theia/core/lib/browser';
+import { QuickPickService } from '@theia/core/lib/browser';
 import { Emitter, Event, Disposable, DisposableCollection } from '@theia/core';
 import { QuickPickItem } from '@theia/core/lib/common/quick-pick-service';
 import { MonacoEditorModel } from '@theia/monaco/lib/browser/monaco-editor-model';
 import { MonacoTextModelService, IReference } from '@theia/monaco/lib/browser/monaco-text-model-service';
 import { OutputUri } from './output-uri';
-import { OutputCommands } from '../browser/output-contribution';
+import { OutputCommands, OutputContribution } from '../browser/output-contribution';
```

Comment:
dependency to browser code from common

Reply:
Thanks for keeping an eye on everything ;) It's a known issue: https://github.com/eclipse-theia/theia/issues/4306

Reply:
Do you want me to move the modules from `common` to `browser`?  I do not mind moving them now, but it's a breaking change. This PR most likely won't make it to the next release anyways.

Reply:
As you want.